examples/citrix_adc/README.md

```diff
@@ -271,19 +272,19 @@ Now that the http services have been create, Ansible has to create a virtual IP
 
 ## Executing the playbook
```

Comment:
If executing the playbook must be done to complete the example, I strongly suggest adding this to the Getting Started section as a step and adding "**Step #:**" here

Reply:
I agree, this can be added to the "Getting Started" section